lib/charms/opensearch/v0/opensearch_users.py

```diff
@@ -313,33 +332,3 @@ def put_internal_user(self, user: str, hashed_pwd: str):
                 COSUser,
                 [{"op": "replace", "path": "/opendistro_security_roles", "value": roles}],
             )
-
-    def _remove_lingering_users(self, relation_users: Set[str]):
```

Comment:
let's rollback

Reply:
Reworked, closing thread. New thread may raise alongside the new solution.